lib/stat_tracker.rb

```diff
@@ -43,6 +43,30 @@ def count_of_teams
     teams.uniq.count
   end
 
+  def best_offense
+    best_offense = get_goals.max_by do |team, stats|
+      stats[:total_goals] / stats[:total_games].to_f
+    end
+  end
+
+  def get_goals
+    team_id_hash = {}
+
+    teams.each do |team|
+      team_id_hash[team['team_id']] = {total_games: 0, total_goals: 0}
+    end
+
```

Comment:
Good work!  I'm glad you finally got this working!
Later on, we'll likely have to break this into helper methods and change some of this to instance variables.  The nested each statements on 59 and 60 may not be ideal, but we can look at this later and refactor somehow.

Overall, fantastic job!

Reply:
Thanks @Stearnzy! I was able to verify this morning that they are passing the spec harness, and thank you for pointing that out!

I agree there may be away to break up those nested loops into helper method and will look into that!

Also, this morning I was able to slightly rework my helper methods and was able to use them for all the other method for the League statistics section so I am done will all of those now. You should see the updates in the files changed section!